backend/src/main/java/lt/vu/tube/entity/Video.java

```diff
@@ -0,0 +1,112 @@
+package lt.vu.tube.entity;
+
+import lt.vu.tube.enums.VideoStatusEnum;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+
+import javax.persistence.*;
+import java.sql.Timestamp;
+import java.util.UUID;
+
+@Entity
+public class Video {
+    @Id
+    @GeneratedValue
+    private UUID id;
+
+    @Column(unique = true)
+    private String path;
+
+    private String fileName;
+
+    private Long fileSize;
+
+    @CreationTimestamp
+    private Timestamp created;
+
+    @UpdateTimestamp //Change this if we add views or some other non meaningful update
+    private Timestamp lastEdited;
+
+    private Boolean isPublic = false;
+
+    private VideoStatusEnum status = VideoStatusEnum.NONE;
+
+    private String mime;
+
+    // TODO: add these whenever we get users
+    //@ManyToOne
+    //private User owner;
+
+
+    public UUID getId() {
```

Comment:
We are not using Lombok? Not saying that we should, just that it was a requirement for first personal task.

Reply:
When I tried using lombok in a jpa entity, it broke it, maybe we should avoid using lombok here 🗡️

Reply:
Totally agree